yarn/src/main/scala/org/apache/spark/deploy/yarn/YarnAllocator.scala

```diff
@@ -94,6 +98,13 @@ private[yarn] class YarnAllocator(
   // Additional memory overhead.
   protected val memoryOverhead: Int = sparkConf.getInt("spark.yarn.executor.memoryOverhead",
     math.max((MEMORY_OVERHEAD_FACTOR * executorMemory).toInt, MEMORY_OVERHEAD_MIN))
+
+  // Maximum number of executor failures per minute 
+  private val relativeMaxExecutorFailurePerMinute = 
+               sparkConf.getInt("spark.yarn.max.executor.failuresPerMinute", -1)
```

Comment:
These should be indented two spaces.  This goes for a couple other places as well.